### ISSUE TITLE
A brief treatise for FTL races/implementations

### ISSUE THREAD

**Issue by kilozombie** [d3athrow/vgstation13#34885]
Hello, gamers. You might have played FTL, the best roguelite in existence. Here's a list of their playable races:

- Humans, which have base stats
- Engis, which repair quickly but deal half damage
- Mantis, which move quickly and attack quickly but repair half as quickly
- Rockmen, who are slow but practically immune to fire
- Zoltans, who provide power ambiently wherever they go
- Slugs, which are psychic and psy-immune
- Lanius, who drain oxygen at an astonishing rate and don't need it to live
- Crystals, who can lock down rooms they're in

I think the vast majority of these only work for FTL instead of our game, SS13. However, a couple of them appeal to me, and so here are my very broad ideas for implementation:

- Engis would make a good roundstart race for non-head roles. As bred non-fighters with a partially cybernetic or even totally mechanical body, they would begin the round PAX-implanted and incapable of fighting. Even if the implant were removed, they are practically useless in physical brawls. In addition, they suffer a tremendous ion and EMP allergy, and would be even more vulnerable than a fully-cybernetic human. In turn, they would be immune to space regardless of clothing--though they can still wear jumpsuits, shoes, and helmets--and would see a speed boost in basic tasks like wrenching, surgery, and even reloading. They make for great subordinates but terrible leaders.

- Mantids are difficult to control at the best of times and are inadequate for many precision tasks. Because they have claw arms instead of hands, they only have **one** hand slot, although with practice they won't see any slowdown in physical tasks. They also cannot wear gloves or shoes, making things like ZAS or shocked doors anathema to them. Finally, because of Nanotrasen's worries about their ability to be controlled, they begin the round loyalty implanted. However, their exceptional speed makes them decent manual laborers, beating out the slowness of a hardsuit, though again not being able to wear magboots makes them primarily useful as Shaft Miners, Xenoarchaeologists, or Paramedics. Due to their clumsy, lanky demeanor, they will **not** be hired as Security Officers, but their toughness, fast healing, and speed make them pretty good Emergency Response Team or Deathsquad members. Many a crew finds themselves horrified to discover they are being rescued, or expunged, by Mantids.

Are these ideas any good? Uhhhh... maybe? They seem incidentally interesting, like the single hand slot and roundstart PAX-implant, but the implementation of either one would take a decent effort and it's not like we're hurting for races. Still, I wanted to throw my hat in the ring.

edit: I have never played FTL13 and know nothing about racial implementation there. These are home-grown /vg/ shit ideas.

[featurerequest] [needspritework] [bugfix]

**Comment by angryturnip** [comment]
No! I want fish in a fish bowl and robot body race!!!!! But serious answer is that I think any new races that get added should be unique and we already have insectoids, so mantids wouldn't really be great as a whole new race. Although if you wanted to add mantis-like insectoid customization options and maybe even a mantis claw sort of round-start "prosthetic" option those could be cool.

**Comment by wotisjanitor** [comment]
> Are these ideas any good?

No

**Comment by DorainGrey** [comment]
Engi would be a cool idea, It'd be nice to have a race really optimized to Engineer but bad at other stuff like fighting etc.
I'd substitute pax implant for Clumsy gene or similar though. I think a weakness to EMP wouldn't work either, given that Engineers are regularly exposed to those.

Mantis I'm not too sure about/

**Comment by Optimism333** [comment]
A round-start robotic race sounds interesting, and it would be neat to have more expansive robotics in general.

No real strong feelings about any of the other ideas, just meh.

**Comment by kilozombie** [comment]
I think a race that can only use one hand slot is pretty interesting. It'd force you to do a ton of micromanagement with containers and make a lot of things take longer by proxy. That's the main gimmick for mantis that I thought was actually worth something but it could easily go on a variety of other bases.

**Comment by Eneocho** [comment]
There's this one MMI in a humanoid metal body race in TG that would kinda fit here, with a couple tweaks, plus it would give a use to ions beyond AI ROGUE or pulse demon fuckery.

**Comment by NigglyWiggly** [comment]
IPCs WHEN

**Comment by Kurfursten** [comment]
"I want a race called Engi that moves through space with no suit, but is vulnerable to EMP" said the man who had never played engineering before
"I want a race to be miner, xenoarch, or paramedic but doesn't have access to magboots" said the man who had never dealt with ZAS before

In seriousness, we have some of these ideas already. A race with weak punches? Grey. Telepathic? Grey or shrrom. No gloves + no shoes + punches deal bonus damage? Catbeast.

**Comment by kilozombie** [comment]
You can ignore ZAS as a paramedic if you believe in yourself.

In all seriousness, as a spitball, it *is* pretty weak. I figured I'd give it a shot though.

**Comment by SonixApache** [comment]
Though I like the idea, and having a bunch of AI mantisfolk walking around the Kestrel shuttle would be fitting, I feel like we've already got half of these races one way or another. Minus a full-cybernetic race, of course.

**Comment by wotisjanitor** [comment]
Said it before but we need more Star Trek-like races. As in side grades to humans. Nothing like this should be playable roundstart but should show up occasionally in vaults or random events. ![image](https://github.com/vgstation-coders/vgstation13/assets/69266620/97c32b0c-9e5b-4fc2-9386-8c28fe6d333e)

**Comment by NigglyWiggly** [comment]
These aren't terrible ideas on their own, but I don't know how people would react to 'this race can do X faster than a human' regardless of how many caveats that comes with. I also don't know how they'd feel about roundstart pax/loyalty implants based on species, but I think that could be interesting

**Comment by pedr69** [comment]
> No gloves + no shoes + punches deal bonus damage? Catbeast.

roundstart tajarans when???
